### PR TITLE
Vendor licenses

### PR DESCRIPTION
**Pull request by sk-arm**
Created vendor license file for legal approval for make the repo public.

### CODE REVIEW

File: vendor-licenses.txt [hunk 1]
```diff
@@ -0,0 +1,1008 @@
+===============================================================================
+This file lists the package level copyright and license information for third 
+party software included in this release of 'edge-proxy'.
```
Comment (ZakariasL) [inline vendor-licenses.txt]: Replace `edge-proxy` with `kube-router`

File: vendor-licenses.txt [hunk 2]
```diff
@@ -0,0 +1,1008 @@
+===============================================================================
+This file lists the package level copyright and license information for third 
+party software included in this release of 'edge-proxy'. 
+
+Refer to the License Agreement that accompanies this release of 'edge-proxy'
```
Comment (ZakariasL) [inline vendor-licenses.txt]: Same as above

**Review by ZakariasL — CHANGES_REQUESTED** [review]

File: vendor-licenses.txt (hunk 1 above)
Comment (sk-arm) [reply vendor-licenses.txt]: Changed to PelionIoT/kube-router.

File: vendor-licenses.txt (hunk 2 above)
Comment (sk-arm) [reply vendor-licenses.txt]: Changed to PelionIoT/kube-router.

**Review by ZakariasL — APPROVED** [review]
LGTM

**Pull request merged**